### PR TITLE
[3304][15.0][MIG] partner_attribute_sst: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3304](https://www.quartile.co/web#view_type=form&model=project.task&id=3304&active_id=3304&menu_id=)

### CODE REVIEW

**Review by kanda999 — APPROVED** [review]
Function review

File: partner_attribute_sst/views/res_occupation_views.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
```
Comment (yostashiro) [inline partner_attribute_sst/views/res_occupation_views.xml]: Shall we get rid of the form view.

File: partner_attribute_sst/views/res_identification_type_views.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
```
Comment (yostashiro) [inline partner_attribute_sst/views/res_identification_type_views.xml]: Shall we get rid of the form view.

File: partner_attribute_sst/views/res_partner_views.xml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                    groups="purchase.group_purchase_user"
+                />
+                <field
+                    name="identification_number"
+                    groups="purchase.group_purchase_user"
+                />
+            </xpath>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="res_partner_kanban_view">
```
Comment (yostashiro) [inline partner_attribute_sst/views/res_partner_views.xml]: @kanda999 Can you please review and check if we should still keep all these changes in the kanban view?

Comment (kanda999) [reply partner_attribute_sst/views/res_partner_views.xml]: @AungKoKoLin1997 
I checked with customer and they said kanban view customize wasn't necessary.

**Comment by kanda999** [comment]
This module is no longer required in sst-custom. In rmm-custom this module is covered by the partner_secondhand_dealer+partner_contact_gender+partner_fax module.

**Pull request closed without merging**